google_takeout_parser/parse_json.py

```diff
@@ -186,15 +193,15 @@ def _parse_semantic_location_history(p: Path) -> Iterator[Res[PlaceVisit]]:
     timelineObjects = json_data.get("timelineObjects", [])
     for timelineObject in timelineObjects:
         if "placeVisit" not in timelineObject:
-            # yield RuntimeError(f"PlaceVisit: no 'placeVisit' key in '{p}'")
+            yield RuntimeError(f"PlaceVisit: no 'placeVisit' key in '{p}'")
```

Comment:
Uncommented this to raise the RuntimeError instead of failing silently. Let me know if this should be left commented out.

Reply:
this one can be commented out, it just leads to too much spam:

```
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
[W 240925 08:57:25 path_dispatch:385] PlaceVisit: no 'placeVisit' key in '/home/sean/.cache/gt/Takeout-New/Location History/Semantic Location History/2023/2023_SEPTEMBER.json'
```

Reply:
got it! I revised it in the latest commit.